Makefile

```diff
@@ -44,3 +44,9 @@ deploy: azd-login ## 🚀 Deploy everything to Azure
 	@azd deploy web --no-prompt
 	@azd deploy function --no-prompt
 	@azd deploy adminweb --no-prompt
+
+docker-build: ## 🐳 Build the Docker image
```

Comment:
What's the need for this? We're not calling it in the workflow, but the fact that it's hardcoded to `fruoccopublic.azurecr.io` seems like we wouldn't want to run it locally either?

Reply:
I've removed these as was just helpful whilst debugging - but we don't call them and you are right nobody knows the key to push them to that registry